app/controllers/sessions_controller.rb

```diff
@@ -0,0 +1,6 @@
+class SessionsController < ApplicationController
```

Comment:
Yes, this is 3-way authentication, and this solution is meant to be a temporary one. 

Both of those sounds good to me - what's a good way to make point 2. happen? (sorry for the silly questions - I'm new to web programming)